### ISSUE TITLE
Error on save when the "trim whitespace on save" setting is enabled

### ISSUE THREAD

**Issue by 2BitSalute** [AllanBlanchard/vscode-ocp-indent#1]
Setup:
- OcpIndent extension is installed
- The setting "Files: Trim Trailing Whitespace" is enabled

Repro:
- In an OCaml source file, add a space at the end of a line
- Save

Expected: no error

Actual:

```
ocp-indent: internal error, uncaught exception:
Sys_error("No such file or directory")

Source: OcpIndent (Extension)
```

![image](https://user-images.githubusercontent.com/23589257/116749826-5d562200-a9b6-11eb-8e54-ea7c3d438b81.png)

Workarounds:
1) Turn off the built-in setting that trims whitespace
2) Install an extension for trimming whitespace

I installed [Trailing Spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces), and it both trims the whitespace and doesn't cause OcpIndent to produce that error.

**Comment by AllanBlanchard** [comment]
This is strange, I have the same option enabled and it does not happen, nor on my regular projects nor in the debug mode of the extension development tool :thinking: .

Have you changed other options in the OcpIndent plugin?

**Comment by AllanBlanchard** [comment]
Should be fixed by #2
